Prometheus version update

prhomhyse opened the pull request:
<!--
Describe the change in summary section, including rationale and degin decisions.
Include "Fixes #nnn" if you are fixing an existing issue.

In "Component Name" section write which component is changed in this PR. This
will help us review your PR quicker.

If you have more information you want to add, write them in "Additional
Information" section. This is usually used to help others understand your
motivation behind this change. A step-by-step reproduction of the problem is
helpful if there is no related issue.
-->

##### Summary
Fixes #6630 

This PR updates the version of Prometheus in the documentation and also fixes some markdown lint warnings.

##### Component Name
doc
##### Additional Information

joelhans commented:
I wonder if we can use one of these one-liners to always download the latest version of the package that begins with `prometheus` and ends with `linux-amd64.tar.gz`:

https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8

5 minutes of experimenting led nowhere for me, but if you can figure it out, that'll help make the article version-agnostic.

prhomhyse commented:
@cakrit 
I have removed the mention of versions in both files
Thanks for the tip @joelhans 
I have figured it out and added it to both files.

And as expected your other PR conflicted and thankfully, it was just one of the files 😅
